doc/rose-rug-task-run.html

```diff
@@ -692,6 +692,114 @@ <h2 id="rose-task-run.util.rose_arch">Built-in Application: rose_arch</h2>
 # ...
 </pre>
 
+    <h2 id="rose-task-run.util.rose_bunch">Built-in Application:rose_bunch</h2>
+
+    <p>This built-in application allows running of multiple command variants in
+    parallel under a single job, as defined by the application
+    configuration.</p>
+
+    <p>The application is normally configured in the
+    <kbd>[bunch-settings]</kbd> and <kbd>[bunch-args]</kbd> sections in
+    <var>rose-app.conf</var>.</p>
+
+    <p>Each variant of the command is run in the same working directory with
+    its output directed to separate <var>.out</var> and <var>.err</var> files
+    of the form <var>bunch.&lt;name&gt;.out</var>. Should you need separate working
+    directories your should configure your command to create the appropriate
+    subdirectory for working in.</p>
+
+    <p>Note that, when running under batch submission systems, you will need
+    to set resource requests to reflect the resources required by running
+    multiple commands at once e.g. if one command would require 1GB memory and
+    you have configured your app to run up to 4 commands at once then you will
+    need to request 4GB of memory.</p>
+
+    <p>The following <kbd>[bunch-settings]</kbd> settings are accepted:</p>
+
+    <dl>
+      <dt><kbd>command-format=FORMAT</kbd></dt>
+
+      <dd>Compulsory. A Pythonic <code>printf</code> style format string to
+      construct the commands to run. Insert placeholders <var>%(ARGNAME)s</var>
+      for substitution of the arguments specified under <kbd>[bunch-args]</kbd>
+      to the invoked command. The placeholder <var>%(instances)s</var> is
+      reserved for inserting an automatically generated index for the command
+      invocation when using the <kbd>instances</kbd> setting.</dd>
+
+      <dt><kbd>instances=N</kbd></dt>
+
+      <dd>Optional. Allows the user to specify an integer value for the number
+      of instances of a command they want to run. This generates the values
+      used by the <var>%(instances)s</var> value in <kbd>command-format</kbd>
+      Useful for cases where the only difference between invocations would be
+      an index number e.g. ensemble members. Note indexes start at 
+      <var>0</var>.</dd>
+
+      <dt><kbd>limit=N</kbd></dt>
+
+      <dd>Optional. Allows the user to limit the number of concurrently running
+      commands. If not specified then all command variations will be run at
+      the same time.</dd>
+
+      <dt><kbd>fail-handle=continue|abort</kbd></dt>
+
+      <dd>Optional (default=continue). Specify what action you want the job to
+      take on failure of a command it is trying to run. If set to
+      <var>continue</var> it will run all command variants . If set to
+      <var>abort</var> then on failure of on of the command variants it will
+      stop trying to run any new variants N.B. the job will wait for any
+      already running commands to finish before exiting.</dd>
+
+      <dt><kbd>incremental=True|False</kbd></dt>
```

Comment:
I prefer this way with the option of being able to configure the app to run in incremental mode or not - you may not want to re-do the file extract/install elements of the app every time you re-run all your commands for instance.